### DISCUSSION TITLE
Use GitHub Discussions in favor for Gitter?

### DISCUSSION THREAD

**Discussion by tomschr** [python-semver/python-semver#330] [Q&A] [answered]
Hi,

some time ago I've created a python-semver channel on [Gitter.im](https://gitter.im/python-semver/community). I'm wondering if we should remove that and establish GH Discussions as one channel.

Any likes or dislikes?
@tlaferriere Thomas, as you suggested Gitter in #282, what do you think? Any preferences? :slightly_smiling_face:

**Comment by tlaferriere** [comment, marked as answer]
That works pretty well even though the resemblance with the issues page is uncanny

**Reply by tomschr** [reply]
Haha, that's true. Well, I'm just thinking out loud. I don't want to restrict the community. On a personal note, I think I prefer more a single site over a project which is distributed over several different sites.
